container/image.bzl

```diff
@@ -123,74 +123,73 @@ def _image_config(
             labels[label] = "@" + label_file_dict[fname[1:]].path
         else:
             labels[label] = fname
-
     args = [
-        "--output=%s" % config.path,
+        "-outputConfig=%s" % config.path,
     ] + [
-        "--manifestoutput=%s" % manifest.path,
+        "-outputManifest=%s" % manifest.path,
     ] + [
-        "--entrypoint=%s" % x
+        "-entrypoint=%s" % x
         for x in entrypoint
     ] + [
-        "--command=%s" % x
+        "-command=%s" % x
         for x in cmd
     ] + [
-        "--ports=%s" % x
+        "-ports=%s" % x
         for x in ctx.attr.ports
     ] + [
-        "--volumes=%s" % x
+        "-volumes=%s" % x
         for x in ctx.attr.volumes
     ] + [
-        "--null_entrypoint=%s" % null_entrypoint,
+        "-nullEntryPoint=%s" % null_entrypoint,
     ] + [
-        "--null_cmd=%s" % null_cmd,
+        "-nullCmd=%s" % null_cmd,
     ]
     if creation_time:
-        args += ["--creation_time=%s" % creation_time]
+        args += ["-creationTime=%s" % creation_time]
     elif ctx.attr.stamp:
         # If stamping is enabled, and the creation_time is not manually defined,
         # default to '{BUILD_TIMESTAMP}'.
-        args += ["--creation_time={BUILD_TIMESTAMP}"]
+        args += ["-creationTime={BUILD_TIMESTAMP}"]
 
-    args += ["--labels=%s" % "=".join([key, value]) for key, value in labels.items()]
-    args += ["--env=%s" % "=".join([
+    args += ["-labels=%s" % "=".join([key, value]) for key, value in labels.items()]
```

Comment:
Try removing the `@` here so that the `extractValue` function becomes unnecessary.